data/src/main/java/pl/gov/mc/protegosafe/data/mapper/ClearMapperImpl.kt

```diff
@@ -0,0 +1,11 @@
+package pl.gov.mc.protegosafe.data.mapper
+
+import com.google.gson.Gson
+import pl.gov.mc.protegosafe.data.model.ClearData
+import pl.gov.mc.protegosafe.domain.model.ClearItem
+import pl.gov.mc.protegosafe.domain.model.ClearMapper
+
+class ClearMapperImpl: ClearMapper {
+    override fun toEntity(json: String): ClearItem =
+        Gson().fromJson(json, ClearData::class.java).toEntity()
```

Comment:
Wydaje mi sie, ze instancje obiektu Gson() mozna by trzymac jako pole prywatne w klasie, zeby nie tworzyc za kazdym razem nowego obiektu.

Reply:
@NieKam dzięki za uwagę, rzeczywiście można by Gsona wyrzucić do DI, weźmiemy to pod uwagę przy najbliższym refaktorze. na ten moment z racji tego że wersja 3.0.2 z tym kodem została już opublikowana pozostawmy ten kod bez zmian